script/install.sh

```diff
@@ -192,6 +195,8 @@ Description
 Options
 
   -a --activate     Only print the path to the activate script
+  --dmd             Only print the path to the compiler binary (e.g. dmd, ldmd2, gdmd)
+  --dc              Only print the path to the compiler binary (e.g. dmd, ldc, gdc)
```

Comment:
* What will happen if I use both `--dmd` and `--dc` together?

* It looks like this documentation could be clearer about the distinction between the flags, i.e. that one is for the DMD-alike compiler binary

Reply:
Yep, conceptually these are actions, not flags.